src/sap.m/src/sap/m/MessageToast.js

```diff
@@ -374,9 +446,32 @@ sap.ui.define(['jquery.sap.global', './InstanceManager', 'sap/ui/core/Popup'],
 			// validate all settings
 			this._validateSettings(mSettings);
 
+			// if content is a string, set as a text control for backward compatibility
+			if (typeof oContent === "string") {
+				oContent = new sap.m.Text({
+					text: oContent,
+					textAlign: "Center"
+				});
+			}
+
+			// group content into a single container control so the popup instance key can be added later
+			oContentContainer = new sap.m.FlexBox({
```

Comment:
- What this is trying to achieve:
I was trying to find a way to link the 'child' controls (those in the content parameter of MessageToast.show) of the toast to the reference of the toast's [popup](https://openui5.hana.ondemand.com/#docs/api/symbols/sap.ui.core.Popup.html), for use by MessageToast.close. This 'link' enables us to close the correct MessageToast popup instance in response to an event of one of the controls within the toast. 

 - Why FlexBox helps achive this:
The FlexBox solution provides a single container to store the reference to the popup as a [custom data property](https://openui5.hana.ondemand.com/#docs/api/symbols/sap.ui.core.Element.html#data). I felt this was the most simple approach I could come up with. Open to ideas though, I don't like having to use a different control just for this but I'm unaware of how else to achieve the link from content to popup. 

_all used modules need to be imported using a require/declare statement_ - would that be [jQuery.sap.require](https://openui5.hana.ondemand.com/#docs/api/symbols/jQuery.sap.html#.require)?

Thanks in advance